### DISCUSSION TITLE
Informazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#778] [Richieste di aiuto]
Buongiorno ,se volessi provare ad integrare una funzione non ancora presente nel registro ,su quale/i codice/i dovrei o potrei agire ?Grazie mille

**Comment by trinko** [comment]
Dipende molto da cosa vuoi fare, ma devi partire dal Controller, che è il codice che gestisce la creazione della pagina che verrà mostrata all'utente. 
I controller sono nella cartella **src/Controller** e sono distinti con un nome che richiama le funzioni principali (Es. AgendaController.php).
Ogni controller è associato all'indirizzo della pagina che trovi indicato nella ROUTE. Ad esempio:
```
#[Route(path: '/agenda/eventi/{mese}'

```

Poi il controller può usare altre funzioni di libreria, che trovi in  **src/Util** o può usare un form definito in **src/Form**.
